ios/RNModules/RNAccountManager/RNAccountManager/RNAccountManager.h

```diff
@@ -9,8 +9,9 @@
 @import Foundation;
 #import "React/RCTBridgeModule.h"
 
+@protocol RCTBridgeModule;
```

Comment:
This shouldn't be necessary - it is defined in the import above

Reply:
Yes, looks like you are right, I will remove it

Reply:
@sjavora Removed